perf.groovy

```diff
@@ -85,26 +85,41 @@ def static getOSGroup(def os) {
 
 // Create the Linux/OSX/CentOS coreclr test leg for debug and release and each scenario
 [true, false].each { isPR ->
-    ['Ubuntu'].each { os ->
+    ['Ubuntu14.04'].each { os ->
         def osGroup = getOSGroup(os)
         def architecture = 'x64'
         def configuration = 'Release'
+		def runType = isPR ? 'private' : 'rolling'
+		def benchViewName = isPR ? 'coreclr private \$ghprbPullTitle' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
         def newJob = job(Utilities.getFullJobName(project, "perf_${os}", isPR)) {
+			
+			label('linux_clr_perf')
+				wrappers {
+					credentialsBinding {
+						string('BV_UPLOAD_SAS_TOKEN', 'CoreCLR Perf BenchView Sas')
+					}
+				}
+			
             steps {
-                shell("sudo bash ./tests/scripts/perf-prep.sh --branch=${projectName}")
-                shell("sudo ./init-tools.sh")
-                shell("""sudo bash ./tests/scripts/run-xunit-perf.sh \\
+                shell("bash ./tests/scripts/perf-prep.sh")
+                shell("./init-tools.sh")
+				shell("./build.sh ${architecture} ${configuration}")
+				shell("GIT_BRANCH_WITHOUT_ORIGIN=\$(echo \$GIT_BRANCH | sed \"s/.*\\/\\(.*\\)/\\1 /\")\n" +
```

Comment:
I think you want to change the sed pattern from  
`s/.*\\/\\(.*\\)/\\1 /\")`  
to  
`s/[^/]*\\/\\(.*\\)/\\1 /\")`  
The former turns origin/dev/foo/bar into just 'bar', while the correct branchname should be dev/foo/bar. This is assuming, of course, that `$GIT_BRANCH` is always prefixed with the remote name?